docs/endpoints.rst

```diff
@@ -840,28 +811,38 @@ Compare length of different types of streets for two or more regions.
              } ]
            }
 
+.. http:post :: /elements/(aggregation)/ratio/groupBy/boundary
 
-.. http:post :: /elements/(aggregation)/density/groupBy/boundary/groupBy/tag
-
-   Get ``density`` of ``aggregation`` of OSM elements grouped by ``boundary`` and ``tag``.
+   Get ``ratio`` of ``aggregation`` of OSM elements grouped by ``boundary``.
    
-   * aggregation type: same as for **/elements/(aggregation)/groupBy/boundary/groupBy/tag**.
-
+   * aggregation type: one of ``area``, ``count``, ``length``, ``perimeter``
+	
+   :query <other>: see above_
+   :query filter2: see filter_
+   :query keys2: Deprecated! see **filter2**
+   :query types2: Deprecated! use **filter2**
+   :query values2: Deprecated! see **filter2**
 
 Users Aggregation
 -----------------
 
 .. http:post :: /users/count
 
-    Compute data aggregation functions on users. Possbile endpoints:
+    Get ``aggregation`` on OSM users. Possible endpoints:
```

Comment:
To _get aggregation on users_ doesn't sound too clear to me. Maybe this would be better:

```suggestion
    Get ``aggregation`` statistics about OSM users. Possible endpoints:
```